openedx/core/djangoapps/theming/helpers_static.py

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I split this out since it was the only thing using `staticfiles.storage` - could probably go back into `helpers.py` since I've created the other `helpers_dirs.py` file.